etl/src/main/java/org/pih/hivmigration/etl/sql/EncounterMigrator.groovy

```diff
@@ -122,6 +122,19 @@ class EncounterMigrator extends SqlMigrator {
               and e.encounter_date is not null   # TODO: figure out what to do with these https://pihemr.atlassian.net/browse/UHM-3237
             ;
         ''')
+
+        executeMysql("Log warnings for encounters with null encounter dates", '''
+            INSERT INTO hivmigration_data_warnings (patient_id, encounter_id, field_name, note)
+            SELECT patient_id,
```

Comment:
I am not sure why, but on my local instance the hivmigration_encounters.patient_id is null for all rows. That's why I joined this table with the hivmigration_patients to get the patient_id.